packages/integration-tests/suites/public-api/withScope/init.js

```diff
@@ -0,0 +1,11 @@
+import * as Sentry from '@sentry/browser';
+
+window.Sentry = Sentry;
```

Comment:
The SDK should be setting `Sentry` on the window automatically - we don’t have to manually do this.

Reply:
Weird, it's not set on the test browser. Maybe Playwright blocks auto assignments to `window`, I'll dig deeper on it.

Reply:
Imo we can just leave it for now to get these tests in!

Something we can come back to.